starfish/image/_filter/richardson_lucy_deconvolution.py

```diff
@@ -6,15 +6,23 @@
 from scipy.signal import convolve, fftconvolve
 
 from starfish.imagestack.imagestack import ImageStack
-from starfish.types import Indices, Number
+from starfish.types import Number
 from ._base import FilterAlgorithmBase
-from .util import gaussian_kernel, preserve_float_range
+from .util import (
+    determine_axes_to_group_by,
+    gaussian_kernel,
+    preserve_float_range
+)
 
 
 class DeconvolvePSF(FilterAlgorithmBase):
 
     def __init__(
-            self, num_iter: int, sigma: Number, clip: bool=True) -> None:
+            self,
```

Comment:
btw, if this fits on one line, you can go ahead and make it all on one line.  my general order of preferences:
1. fit on one line.
2. fit on two lines.
3. one param per line.